sdk/test-utils/recorder-new/src/utils/env.browser.ts

```diff
@@ -3,4 +3,4 @@
 
 // In the browser, we load the env variables with the help of karma.conf.js
 
-export const env = (window as any).__env__;
+export const env = ((window as unknown) as { __env__: typeof process.env }).__env__;
```

Comment:
what is the type of `window`? perhaps it is compatible with `typeof process.env` so I wonder if we can cast it directly to that target type instead of going through casting to `unknown` first.

Reply:
It's not compatible unfortunately, hence the ugly casts :(